### PR TITLE
VerticalDivider component

### PR DESCRIPTION
**Pull request by jcmustin**
Added VerticalDivider component to Container.
screen shot 2019-02-19 at 3 21 39 pm

### CODE REVIEW

File: src/shared-components/container/index.js [hunk 1]
```diff
@@ -4,11 +4,12 @@ import {
   Container,
   Section,
   Divider,
-  Image,
+  Image, VerticalDivider,
```
Comment (snags88) [inline src/shared-components/container/index.js]: Need to move this to a new line. Weird that eslint didn't catch it.

**Comment by jcmustin** [comment]
> Could you update the storybook example so that we can see what a container looks like with the divider?

@snags88 Oh, gosh, of course, sorry! I took screenshots for just that purpose and then just forgot to attach them. I've added the output of the snapshotted component, which includes a vertical divider at the bottom.

File: src/shared-components/container/index.js (hunk 1 above)
Comment (jcmustin) [reply src/shared-components/container/index.js]: That is bizarre; I feel like I did put it on its own line originally, but I must be mistaken. Fixed in fc3cce7f0a2862631ffa28c2fff08b64155cbb5e.

**Comment by snags88** [comment]
@jcmustin Oh I meant in the actual code base under the `stories` directory so that we can see what the vertical divider looks like when we go to http://radiance-ui.curology.com

**Comment by jcmustin** [comment]
Closing this, as changes were made to the design that removed need for this component.

**Pull request closed without merging**